kombu/utils/__init__.py

```diff
@@ -451,3 +452,7 @@ def maybe_fileno(f):
         return fileno(f)
     except FILENO_ERRORS:
         pass
+
+
+def maybe_s_to_ms(v):
```

Comment:
Note that in master the utils package has been refactored into many subpackages instead, and I fear you need to
add a .utils.time at this point.